bin/index.js

```diff
@@ -1,9 +1,18 @@
 #!/usr/bin/env node
 const program = require('commander');
+const didYouMean = require('didyoumean');
 const action = require('../lib/action.js');
 
 program.version(require('../package.json').version);
 
+const suggestCommands = (cmd) => {
+	const suggestion = didYouMean(cmd, program.commands.map(cmd => cmd._name));
+	if (suggestion) {
+		console.log();
+		console.log(`Did you mean ${suggestion}?`);
+	}
+};
+	
```

Comment:
Hi James, This would be really helpful and the code is looking great! Would you mind moving this function to `lib/helper.js`? I think it would be better to keep `index.js` as plain as possible.